content/chapter-sources/webhook/_index.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Webhook
+linktitle: "webhook() source"
+weight: 6000
+driver: "webhook(), webhook-json()"
+short_description: "Receive logs via a HTTP webhook"
+---
+<!-- This file is under the copyright of Axoflow, and licensed under Apache License 2.0, except for using the Axoflow and AxoSyslog trademarks. -->
+
+Starting with version 4.8.0, {{% param "product_name" %}} can collect logs via a webhook using the `webhook()` and `webhook-json()` sources. The `webhook-json()` source automatically parses the payload using the [`json-parser()`]({{< relref "/chapter-parsers/json-parser/_index.md" >}}).
+
+Example minimal config:
+
+```shell
+source s_webhook {
+webhook-json(
+    port(8181)
+    paths(["/events","/events/(?P<HOST>.*)"])
+    );
+};
+```
+
+To test the source, you can use `curl` for example, on the host where {{< product >}} is running:
+
+```shell
+curl -X POST --data "{'MESSAGE':'message-value'}" http://127.0.0.1:8181/events
+```
+
```

Comment:
Let's mention that if query parameters are set by the sender (e.g. `?category=shoes&color=blue&size=9`), these will be available under the `webhook.query.*` (`${webhook.query.color}`, `${webhook.query.size}` macros.